ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -604,6 +604,8 @@ module.exports = Marionette.LayoutView.extend({
               color,
             })
             this.map.showCircleShape(locationModel)
+          } else if (CQLUtils.isLineFilter(value)) {
+            this.handleFIlterAsLine(filter, color)
```

Comment:
✏️ Looks like you've got handleFIlter (capital i in filter) instead of handlerFilter